Code/Framework/AzToolsFramework/AzToolsFramework/Application/ToolsApplication.cpp

```diff
@@ -15,6 +15,8 @@
 #include <AzCore/Debug/Profiler.h>
 #include <AzCore/Settings/SettingsRegistryMergeUtils.h>
 
+#include <AzFramework/AzFrameworkNativeUIModule.h>
+#include <AzFramework/Components/NativeUISystemComponent.h>
```

Comment:
The `NativeUISystemComponent.h` wouldn't be need in this file, if the `AzFrameworkNativeUiModule` class overrides the `AZ::Module::GetRequiredSystemComponents` function to return the NativeUiSystemComponent as a required component.